webextensions/api/tabs.json

```diff
@@ -2362,7 +2356,10 @@
                 "version_added": "54",
                 "notes": "Available for use in Manifest V2 only."
               },
-              "opera": "mirror",
+              "opera": {
```

Comment:
@queengooborg Where does this come from?

Reply:
This came from mirroring from Chrome, just with dropping the range denominator to satisfy the consistency linter!